README.md

```diff
@@ -56,60 +56,15 @@ scoop install python ruby go perl
 
 If you've built software that you'd like others to use, Scoop is an alternative to building an installer (e.g. MSI or InnoSetup) — you just need to zip your program and provide a JSON manifest that describes how to install it.
 
-## Requirements
-
-- Windows 7 SP1+ / Windows Server 2008+
-- [PowerShell 5](https://aka.ms/wmf5download) (or later, include [PowerShell Core](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-6)) and [.NET Framework 4.5](https://www.microsoft.com/net/download) (or later)
-- PowerShell must be enabled for your user account e.g. `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
-
```

Comment:
Why removes these requirements?

BTW, the PowerShell version should be written as:

```markdown
- [Windows PowerShell 5.1](https://aka.ms/wmf5download) (or [PowerShell](https://aka.ms/powershell))
- [.NET Framework 4.5](https://www.microsoft.com/net/download) or later
```

Reply:
The standalone installer's readme stated these requirements. I'm trying to simplify the instruction here so we don't have to maintain different installation instruction copies. Users should be able to find the complete instruction via [the link here](https://github.com/ScoopInstaller/Scoop/pull/4825/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R67). I would update the requirements per your suggestion to https://github.com/ScoopInstaller/Install#prerequisites and the website.